src/components/Modal/ModalSkeleton.js

```diff
@@ -54,6 +43,7 @@ const Modal = (props) => {
         KeyboardShortcut.unsubscribe('Enter');
     };
     const {
+        useNativeDriver,
```

Comment:
I would maybe remove this. We can make it so all Android modal `useNativeDriver` and this would keep things a lot simpler.

Reply:
Ok, So there are multiple types of modal. IF we make a true in the `index.android.js` then it will be true for all the types of the modal. Do you want that? If so, We can follow your next comment and remove the `useNativeDriver` from the `getModalStyles.js`.

Reply:
Yes